src/pybind/mgr/dashboard/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ set(mgr-dashboard-nodeenv ${CMAKE_CURRENT_BINARY_DIR}/node-env)
 
 add_custom_command(
   OUTPUT "${mgr-dashboard-nodeenv}/bin/npm"
-  COMMAND ${CMAKE_SOURCE_DIR}/src/tools/setup-virtualenv.sh --python=python2.7 ${mgr-dashboard-nodeenv}
+  COMMAND ${CMAKE_SOURCE_DIR}/src/tools/setup-virtualenv.sh --python=python${MGR_PYTHON_VERSION} ${mgr-dashboard-nodeenv}
```

Comment:
could just use `${MGR_PYTHON_EXECUTABLE}` here instead of `python${MGR_PYTHON_VERSION}`